proposals/20200913-rootlessKubeflow.md

```diff
@@ -0,0 +1,35 @@
+# Rootless Kubeflow
+
+### Goals
+
+We want to run Kubeflow 99 % rootless accoring to CNCF/Kubernetes best practices.
+
+The main steps are adding an additional profile for istio-cni or ambient mesh, updating the documentation and manifest generation process. 
+Then adding the baseline and restricted PSS as kustomize component to /contrib and extending the profile controller to annotate user namespaces with configurable PSS labels.
```

Comment:
Could you expand more on why additional profile for istio-cni or ambient mesh is necessary in addition to the PSS set